apps/prairielearn/elements/pl-big-o-input/pl-big-o-input.py

```diff
@@ -82,8 +82,10 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
             phs.convert_string_to_sympy(
                 a_true, variables, allow_complex=False, allow_trig_functions=False
             )
-        except phs.BaseSympyError:
-            raise ValueError(f'Parsing correct answer "{a_true}" for "{name}" failed.')
+        except phs.BaseSympyError as err:
+            raise ValueError(
+                f'Parsing correct answer "{a_true}" for "{name}" failed.'
+            ) from err
```

Comment:
Very much a nit: both the docs for the lint rule here (https://docs.astral.sh/ruff/rules/raise-without-from-inside-except/) and the Python docs (https://docs.python.org/3/reference/simple_stmts.html#the-raise-statement) use `exc` as the name for this, and other code in this PR uses `e` instead. It'd be great to pick one of these and try to stick with it. I'm bringing this up only because this is new to our codebase and we don't yet have any conventions here.

Reply:
Interesting, I'll use `exc`. Never seen this before in python code, but also never use the `from err` pattern a lot.